plugins/extractors/bigquery/bigquery.go

```diff
@@ -366,6 +365,10 @@ func (e *Extractor) buildColumn(ctx context.Context, field *bigquery.FieldSchema
 		Attributes:  utils.TryParseMapToProto(attributesMap),
 	}
 
+	if field.Schema != nil {
```

Comment:
`field.Schema` is of the following type:

```go
// Schema describes the fields in a table or query result.
type Schema []*FieldSchema
```

So check for `len(field.Schema) > 0` instead of the `nil` check.

Reply:
Done